src/metabase/lib/drill_thru/automatic_insights.cljc

```diff
@@ -26,7 +26,9 @@
              ;; Column with no value is not allowed - that's a column header click. Other combinations are allowed.
              (or (not column) (some? value))
              (lib.metadata/setting query :enable-xrays)
-             (not-empty dimensions))
+             (not-empty dimensions)
+             ;; TODO fix this drill to work with underlying aggregations and remove this check (metabase#46932).
+             (not (lib.drill-thru.common/strictly-underyling-aggregation? query column)))
```

Comment:
Typo: `underlying`.

Reply:
wow, my fingers do not know how to type "underlying". I made this typo in like 10 different places. most are the result of auto-complete on the original typo, no doubt, but still...